zorg/buildbot/builders/annotated/premerge/dispatch_job.py

```diff
@@ -94,6 +99,10 @@ def start_build_linux(commit_sha: str, k8s_client) -> str:
         "export CC=clang",
         "export CXX=clang++",
         "export POSTCOMMIT_CI=1",
+        f"export SCCACHE_GCS_BUCKET={bucket_name}",
```

Comment:
Don't you need a '$' in front of the '{' ?

Reply:
Python fstrings don't need a `$` in front of the substitution. It would just appear as a $ in the resulting string